postgres/check.py

```diff
@@ -716,6 +716,8 @@ def _get_custom_metrics(self, custom_metrics, key):
     def check(self, instance):
         host = instance.get('host', '')
         port = instance.get('port', '')
+        if port != '':
+            port = int(port)
```

Comment:
Also, this patch would make your ansible role work with this integration, which it doesn't right now...